response_types.go

```diff
@@ -50,6 +50,11 @@ var (
 
 type Evidence map[string]interface{}
 
+type Reason struct {
+	Code   string
+	Source string
```

Comment:
Do we need to specify the JSON tag? Will this be mapped to `code` and `source` or `Code` and `Source`?

Reply:
code and source, i undid it in the https://github.com/inloco/incognia-go/pull/37/commits/7ffc97c153c299c49f5b5b72b3ec212d8770c7bd